ruleset/testing/tests/auditd.ini

```diff
@@ -1,21 +1,200 @@
+log 1 pass = type=ANOM_LOGIN_FAILURES msg=audit(1450770603.209:3300446): Text
+rule = 80723
+alert = 5
+decoder = auditd
+
+[Auditd: Login attempt from a forbidden location.]
+log 1 pass = type=ANOM_LOGIN_LOCATION msg=audit(1450770603.209:3300446): Text
+rule = 80724
+alert = 5
+decoder = auditd
+
+[Auditd: Login attempt reached the maximum amount of concurrent sessions.]
+log 1 pass = type=ANOM_LOGIN_SESSIONS msg=audit(1450770603.209:3300446): Text
+rule = 80725
+alert = 4
+decoder = auditd
+
+[Auditd: Login attempt is made at a time when it is prevented.]
+log 1 pass = type=ANOM_LOGIN_TIME msg=audit(1450770603.209:3300446): Text
+rule = 80726
+alert = 5
+decoder = auditd
+
+[Auditd: SELinux permission check.]
+log 1 pass = type=AVC msg=audit(1226270358.848:238): avc:  denied  { write } for  pid=13349 comm="certwatch" name="cache" dev=dm-0 ino=218171 scontext=system_u:system_r:certwatch_t:s0 tcontext=system_u:object_r:var_t:s0 tclass=dir
+rule = 80730
+alert = 3
+decoder = auditd
+
+[Auditd: SELinux mode (enforcing, permissive, off) is changed.]
+log 1 pass = type=MAC_STATUS msg=audit(1336836093.835:406): enforcing=1 old_enforcing=0 auid=0 ses=2
+rule = 80731
+alert = 10
+decoder = auditd
+
+[Auditd: SELinux error.]
+log 1 pass = type=SELINUX_ERR msg=audit(1311948547.151:138): op=security_compute_av reason=bounds scontext=system_u:system_r:anon_webapp_t:s0-s0:c0,c100,c200 tcontext=system_u:object_r:security_t:s0 tclass=dir perms=ioctl,read,lock
+log 1 pass = type=USER_SELINUX_ERR msg=audit(1311948547.151:138): Text
+rule = 80732
+alert = 10
+decoder = auditd
+
+[Auditd: Replay attack detected.]
+log 1 pass = type=CRYPTO_REPLAY_USER msg=audit(1234567890.123:1234): Text
+rule = 80740
+alert = 12
+decoder = auditd
+
+[Auditd: Group ID changed.]
+log 1 pass = type=CHGRP_ID msg=audit(1450770603.209:3300446): Text
+rule = 80741
+alert = 5
+decoder = auditd
+
+[Auditd: User ID changed.]
+log 1 pass = type=CHUSER_ID msg=audit(1450770603.209:3300446): Text
+rule = 80742
+alert = 5
+decoder = auditd
+
+[Audit: Created: $(audit.file.name).]
+log 1 pass = type=SYSCALL msg=audit(1479982525.380:50): arch=c000003e syscall=2 success=yes exit=3 a0=7ffedc40d83b a1=941 a2=1b6 a3=7ffedc40cce0 items=2 ppid=432 pid=3333 auid=0 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=pts0 ses=2 comm="touch" exe="/bin/touch" key="audit-wazuh-w" type=CWD msg=audit(1479982525.380:50):  cwd="/var/log/audit" type=PATH msg=audit(1479982525.380:50): item=0 name="/var/log/audit/tmp_directory1/" inode=399849 dev=ca:02 mode=040755 ouid=0 ogid=0 rdev=00:00 nametype=PARENT type=PATH msg=audit(1479982525.380:50): item=1 name="/var/log/audit/tmp_directory1/malware.py" inode=399852 dev=ca:02 mode=0100644 ouid=0 ogid=0 rdev=00:00 nametype=CREATE type=PROCTITLE msg=audit(1479982525.380:50): proctitle=746F756368002F7661722F6C6F672F61756469742F746D705F6469726563746F7279312F6D616C776172652E7079
+rule = 80790
+alert = 3
+decoder = auditd
+
+[Audit: Passwd was used to lock an account.]
```

Comment:
same name than 80794? should be different names per test?

Reply:
80794 is a different name, Audit: Passwd was used to **unlock** an account.